pyproject.toml

```diff
@@ -48,6 +48,7 @@ dependencies = [
     "sqlalchemy>=1.4.27,<= 1.5.0",
     "pyyaml>=5.1.2,<7.0",
     "importlib_metadata; python_version < '3.10'",
+    "neurokit2>=0.1.7"
```

Comment:
It's better to have `neurokit2` as optional dependency as it has incompatibilities with `junifer` in some mutual packages like `pandas`.

Reply:
Agreed.

Reply:
For the optional dependency block, I have two suggestions which will work like so:
- `pip install junifer[physio]` : to take care of future uses of `neurokit2`
- `pip install junifer[complexity]` : to make it direct

Let me know your thoughts @omidvarnia @fraimondo.

Reply:
`neurokit2`